<launch>
    <arg name="map"/>
    <node name="rvizvisualisation" pkg="rviz" type="rviz" output="log" args="-d $(find nbv_mapping)/rviz/rviz_config.rviz" />
    <node name="planner_node" pkg="nbv_mapping" type="planner_node" output="screen">
      <param name="map_file" value="$(find nbv_mapping)/config/$(arg map).txt"/>
      <param name="cube_length" value="1"/>
    </node>
</launch>
